index.html

```diff
@@ -2326,6 +2327,107 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="comment">
+			<rdef>comment</rdef>
+			<div class="role-description">
+				<p>A comment contains content expressing reaction to other content.</p>
+				<p>Comments can annotate any visible content, from small spans of text, to other comments, to entire articles. Authors SHOULD identify the relationships between comments and the commented content, as follows:</p>
+				<ol>
+					<li>If the comment is a reply to another <code>comment</code>:
+						<ul>
+							<li>If all ancestor comments are available in the DOM, make the reply <code>comment</code> a semantic descendant of the parent <code>comment</code> being replied to, either as a DOM descendant element, or via <pref>aria-owns</pref>.</li>
```

Comment:
If the reply has a parent, it is already a descendant. Recommend removing the word "parent" and also fix dangling preposition:

```suggestion
							<li>If all ancestor comments are available in the DOM, make each reply <code>comment</code> a semantic descendant of the <code>comment</code> to which it is replying, either by making it a DOM descendant element or by using <pref>aria-owns</pref>.</li>
```

Reply:
I like having the word parent in there, it makes the sentence easier to read, since we're talking about 2 different comments here. I'll take the rest.

Reply:
Ok, I see, you are using "to which it is replying" instead of "parent comment". That means the same thing to me but is longer. For example, when using something like "reddit" there are buttons that say things like "go to parent comment".

Reply:
Wasn't sure I liked the wordiness, but, I went with this in the end.